### DISCUSSION TITLE
Grafana ed Homeassistant in https

### DISCUSSION THREAD

**Discussion by vinny74man** [indomus/forum#1588] [HUB: Home Assistant]
Ciao a tutti come da titolo sto cercando di integrare grafana con homeassistant, avendo homeassistant in https quando vado ad inserire il link del pannello grafana nel card web di homeassistant mi dice che se HA è in https non posso inserire pagine http, quindi ho configurato grafana per andare a prendere gli stessi certificati di HA, se mi collego direttamente a grafana da web "https://miodominio:3000xxxxxx" riesco ad accedere al pannello senza problemi, ma quando inserisco lo stesso link sulla card web di HA mi dice che la pagina è bloccata o non esiste.

Qualcuno ha avuto lo stesso problema e può indicarmi una solzione?
Grazie

HA Core 2022.3.5 installato in venv

**Comment by vinny74man** [comment]
Ciao a tutti nessuno riesce ad aiutarmi sto spulciando un po tutti i post relativi a questo problema ma con nessuno sono riuscito a risolvere, ripeto che collegandomi direttamante all'indirizzo https://miodominio:3000 riesco ad accedere tranquillamente ai grafici, mettendo lo stesso indirizzo in iframe questo risulta essere irrangiungibile o bloccato.
Grazie

**Reply by snakuzzo** [reply]
"irraggiungibile o bloccato" non ha alcun significato.
Devi portare delle evidenze...come ad esempio gli errori che ti mostra il browser in console

**Reply by vinny74man** [reply]
Sono riuscito a trovare la soluzione, bisogna abilitare `allow_embedding = true` nella sezione security del file grafana.ini, questo permette la vusualizzazione dei grafici in iframe, cosa altrimenti non possibile me motivi di sicurezza da parte di grafana.

Risolto questo adesso ho quest'altro problema, quello che si vede negli iframe di HA è si il grafico ma anche la possibilità di modificarlo o di creare altri grafici, c'è modo di bloccare la visualizzazione al solo grafico§?

Grazie
